Add examples for issue 4658 in examples.txt

kevinbouti opened the pull request:
Add Product Role examples (UNCL codes & Digital Twin) for issue #4658

fthobe commented:
@kevinbouti I think this goes into order / cart and not into product. It's a document definition standard and not a product definition standard.

kevinbouti commented:
@fthobe You are technically correct that UNCL 1001 is a document code standard. However, in the context of Digital Product Passports (DPP) and circular economy, we often need to define the intended role of a product (e.g., is this item a "Spare Part" or a "Consumable"?) at the product definition level, long before it enters a cart or order.

To address your concern, I have updated the examples to:

Use DefinedTerm with a generic name (e.g., "Consumable") as the primary value.

Treat UNCL 1001 only as a reference mapping inside inDefinedTermSet, rather than implying the product is a document.

Added non-EDI examples like "Digital Twin" and "Software Component" to show broader usage.

This strikes a balance between semantic strictness and real-world B2B needs

MatthiasWiesmann reviewed (changes requested):
We cannot add examples before the schema changes.

kevinbouti commented on data/ext/pending/issue-4658-examples.txt:
```diff
@@ -0,0 +1,75 @@
+      "category": "Consumable",
+      "additionalProperty": [
+        {
+          "@type": "PropertyValue",
```
Thanks for the guidance. You are right, using "UN/CEFACT Code" as the name was semantically weak.

I have updated the example to use the standard PropertyValue pattern correctly:

Used propertyID to explicitly link to the UN/CEFACT standard URI (machine-readable).

Changed name to a generic "Product Type Code".

Kept the value "711" (Consumable).

fthobe replied:
This doesn't change that the vocabulary is used wrong. Neither GS1 nor CEFACT cover your use case.

kevinbouti reviewed (commented):
have updated the example to use the standard PropertyValue pattern correctly:

Renamed property: Changed name to a generic "Product Type Code" instead of the standard's name.

Added propertyID: Explicitly linked to the UN/CEFACT standard URI (https://service.unece.org/trade/untdid/d19b/tred/tred1001.htm) to make it machine-readable.

fthobe commented on data/ext/pending/issue-4658-examples.txt:
```diff
@@ -0,0 +1,176 @@
+  <div itemprop="additionalProperty" itemscope itemtype="https://schema.org/PropertyValue">
+    <span itemprop="name">Product Type Code</span>
+    <span itemprop="description">UN/CEFACT UNCL 1001 code for Consumable</span>
+    <meta itemprop="propertyID" content="https://service.unece.org/trade/untdid/d19b/tred/tred1001.htm">
```
@MatthiasWiesmann sorry, but this has nothing to do here, CEFACT is not an item categorisation standard, it shouldn't be an example here. You absolutely correctly brought alternative examples for something that still, seems basically arbitrarily adding a field to conform with the desired workflow of one company. We had some really good PRs busted here for much less inconsistency. There are absolutely valid fields for this such as category and keywords which can fully model this. If for any reason (that goes absolutely beyond anything I can comprehend) this goes in, at least remove false application of current standards.
